resources/assets/js/views/EnterPassword.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <div class="page">
+        <p>
+            Enter the password you wish to share and choose for how long it should be available,
+            as well as for how many times it should be retrievable. After that you will receive
+            an URL that you can distribute.
+        </p>
+
+        <div class="input-group input-group-lg bottom-margin">
+            <span for="password-field" class="input-group-addon glyphicon glyphicon-lock" id="sizing-addon1"></span>
+            <input v-model="passwordField" aria-label="Password" type="password"
+                   id="password-field" class="form-control" placeholder="Password" aria-describedby="sizing-addon1">
+        </div>
+
+        <div class="input-group input-group-lg bottom-margin">
+            <span for="password-confirm-field" class="input-group-addon glyphicon glyphicon-lock" id="sizing-addon2"></span>
+            <input v-model="passwordConfirmField" aria-label="Confirm password" type="password"
+                   id="password-confirm-field" class="form-control" placeholder="Confirm Password" aria-describedby="sizing-addon2">
+        </div>
+
+        <div class="bottom-margin">
+            <nof-days-selector @changed="daysChanged"></nof-days-selector>
+            <nof-hours-selector @changed="hoursChanged"></nof-hours-selector>
+        </div>
+
+        <div class="bottom-margin">
+            <nof-views-selector @changed="viewsChanged"></nof-views-selector>
+        </div>
+
+        <p class="btn-container">
+            <a class="btn btn-primary btn-lg" @click="storeSecret" id="get-url-btn" href="#" role="button">
+                Get URL
+            </a>
+        </p>
+    </div>
+</template>
+
+
+<script>
+    import NofViewsSelector from "../components/NofViewsSelector.vue";
```

Comment:
I think this can be shortened to `from "../components/NofViewsSelector";` but I may be wrong